RocketReserver/LaunchListView.swift

```diff
@@ -25,6 +25,9 @@ struct LaunchListView: View {
       .refreshable {
         await viewModel.refresh()
       }
+      .task {
+        await viewModel.fetch()
+      }
```

Comment:
Updating this as `main` of ApolloPagination now supports non-async initialization of `AsyncGraphQLQueryPager`.